GitCommands/RevisionReader.cs

```diff
@@ -251,48 +254,43 @@ private bool TryParseRevision(in ArraySegment<byte> chunk, [NotNullWhen(returnVa
 
             // Next we have zero or more parent IDs separated by ' ' and terminated by '\n'
             int noParents = CountParents(in array, offset);
-            if (noParents < 0)
-            {
-                ParseAssert($"Log parse error, {noParents} no of parents for {objectId}");
-                revision = default;
-                return false;
```

Comment:
Is it correct to not exit on error?

Reply:
any parse error will probably make the parsing to fail.
In this case the check on line 285 should be changed (and something else will fail later)